packages/driver/cypress/integration/e2e/multi-domain/commands/multi_domain_actions.spec.ts

```diff
@@ -55,32 +55,29 @@ context('multi-domain actions', { experimentalSessionSupport: true, experimental
     cy.get('a[data-cy="dom-link"]').click()
 
     cy.switchToDomain('foobar.com', done, () => {
-      const $btn = cy.$$('#button')
```

Comment:
This is a synchronous function and has issues because the commands are now run before the window is loaded.

Reply:
Is this a deviation from how the primary domain runs?

Reply:
it is not. I tested this withe the primary domain and it exhibits the same behaviour.